gazebo_plugins/scripts/wheel_slip_drawbar_pull_publisher.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Desc: helper script for publishing drawbar pull wrenches
+# Author: Steve Peters
+#
+import argparse
+import math
+import rclpy
+import sys
+from geometry_msgs.msg import Wrench
+from rclpy.node import Node
+
+kDefaultTopic = '/drawbar_pull'
```

Comment:
I recommend dropping the forward slash so that the topic name may be pushed into a namespace.
```suggestion
kDefaultTopic = 'drawbar_pull'
```

Reply:
https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1364/commits/dcca4afd7c4876d96efd84e1c117a926f973b242